server/tools/build-keycloak.sh

```diff
@@ -87,14 +87,19 @@ rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
 bin/jboss-cli.sh --file=/opt/jboss/tools/cli/standalone-ha-configuration.cli
 rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
 
+###########
+# Garbage #
+###########
+
+rm -rf /opt/jboss/keycloak/standalone/tmp/auth
+rm -rf /opt/jboss/keycloak/domain/tmp/auth
+
 ###################
 # Set permissions #
 ###################
 
-echo "jboss:x:1000:jboss" >> /etc/group
-echo "jboss:x:1000:1000:JBoss user:/opt/jboss:/sbin/nologin" >> /etc/passwd
-chown -R jboss:jboss /opt/jboss
-chmod -R g+rw /opt/jboss
-
-rm -rf /opt/jboss/keycloak/standalone/tmp/auth
-rm -rf /opt/jboss/keycloak/domain/tmp/auth
+echo "jboss:x:0:root" >> /etc/group
```

Comment:
If the jboss user is going to be in the root group we no longer need to create a jboss group and the group id 0 is already assigned to the root group.

Reply:
This would add the jboss user to the root group: 
sed -e '1s/$/jboss/' /etc/group

Reply:
This line creates a group named 'jboss', with the same group id as the 'root' group, with one group member and that member is 'root'. The jboss user isn't a member of the root group.

Reply:
Since the jboss user is assigned the root group from /etc/passwd we should probably just delete this line altogether.

Reply:
After a bit of digging it seems like the jboss group is simply ignored. Executing 'chgrp jboss aFile' will cause aFile to be in the 'root' group.